docs/admin/catalog.md

```diff
@@ -0,0 +1,92 @@
+Introduction
+------------
+
+The MasterMind Service Catalog is a repository where the Service Types MasterMind can deploy are stored. This includes the deployment template for the Service as well as its configuration template.
+
+This section will describe the structure of the Catalog, and how to extend it if needed.
+
+Catalog Entries
+---------------
+
+The Catalog itself is a repository containing one to many entries. Each entry is a folder, named after the Service Type the entry represents, which contains as a minimum requirement:
+
+  - A docker-compose.yml file which describes the deployment of the IoT Service as a Docker Stack on Docker Swarm
+  - A mastermind.yml file, which describes the Service and its configuration options. This is what MasterMind uses to generate the UI forms to create and deploy new Services.
+
+In addition, depending on the contents of the docker compose file, each entry can also come with:
+
+  - Config files, to be used to create Docker configs
+  - Secret files, to be used to create Docker secrets
+
+Deployment Template
+-------------------
+
+The deployment template for a Catalog entry comes in the form of a docker-compose.yml file, which describes how to deploy the Service on a Docker Swarm cluster. These files should be compliant with Docker Compose Version 3 in order to be used by MasterMind, however not all options available in Compose are currently supported. For a list of supported options see *LINK NEEDED*
```

Comment:
IIRC, it's version 3.3 to support configs in [compose](https://github.com/docker/compose/releases/)